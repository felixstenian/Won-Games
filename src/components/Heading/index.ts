export { default } from './Heading'
export type { HeadingProps, LineColors } from './Heading'
