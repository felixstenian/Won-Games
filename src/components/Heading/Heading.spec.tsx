import { screen } from '@testing-library/react'
import 'jest-styled-components'

import { renderWithTheme } from 'utils'

import Heading from '.'

describe('<Heading />', () => {
  it('1. should render the reading', () => {
    const { container } = renderWithTheme(<Heading>Heading</Heading>)

    expect(screen.getAllByText(/Heading/i))

    expect(container.firstChild).toMatchSnapshot()
  })
})
