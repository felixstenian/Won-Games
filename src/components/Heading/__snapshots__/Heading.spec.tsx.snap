// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Heading /> 1. should render the reading 1`] = `
.c0 {
  font-size: 2.0rem;
  color: #030517;
}

@media (min-width:768px) {
  .c0 {
    font-size: 2.8rem;
  }
}

<h2
  class="c0"
  color="black"
>
  Heading
</h2>
`;
